Documentation/index.rst

```diff
@@ -53,9 +53,22 @@ get started and experiment with Cilium.
    :maxdepth: 2
    :caption: Getting Started
 
-   gettingstarted/index
-   concepts/index
-   gettinghelp
+   gettingstarted/k8s-install-default
+   gettingstarted/hubble_intro
+   gettingstarted/hubble_setup
+   gettingstarted/hubble-configuration
+   gettingstarted/hubble
+   gettingstarted/hubble_cli.rst
+   gettingstarted/terminology
+   gettingstarted/gettinghelp
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Advanced Installation
+
+   advancedinstallation/taints
+   advancedinstallation/index
```

Comment:
Should `taints` and `index` be swapped here? Or should even `taints` go into a dedicated section in `advancedinstallation/index`?